core/src/main/java/eu/tsystems/mms/tic/testframework/report/TesterraListener.java

```diff
@@ -484,6 +483,9 @@ public void onDataProviderFailure(ITestNGMethod testNGMethod, ITestContext testC
             } else {
                 methodContext.addError(exception);
             }
+            // A crashed dataprovider method as a kind of setup occurs a skipped status at the test method
```

Comment:
"[D]ata[P]rovider" as per `@DataProvider` or "data[P]rovider" as per `@Test#dataProvider`.
... I fear I do not understand the rest of the sentence, but I am sure _something_ is wrong. Please have a second look.

Reply:
Updated comment.